README.md

```diff
@@ -14,7 +14,17 @@ The AdoptOpenJDK API provides a way to consume JSON information about the AdoptO
 
 ## Usage
 
-The api is documented via swagger.  The swagger documentation can be viewed at: [swagger-ui](https://api.adoptopenjdk.net/swagger-ui). 
+The API is documented via swagger.  The swagger documentation can be viewed at: [swagger-ui](https://api.adoptopenjdk.net/swagger-ui). 
 The open api definition for this can be viewed at [openapi](https://api.adoptopenjdk.net/openapi).
 
 For more information, including example queries, please look at [STRUCTURE.md](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/docs/STRUCTURE.md)
+
+## Who's using the AdoptOpenJDK API?
+
+There are various consumers of the AdoptOpenJDK API, from individuals to organisations. 
+
+The following list highlights the key consumers and their use-cases:
```

Comment:
Probably need to say this is a small subset, and perhaps provide a link to our dash.adoptopenjdk.net showing the 200M+ downloads that have been served via hte API